src/ol_infrastructure/lib/aws/ec2_helper.py

```diff
@@ -173,3 +207,53 @@ def route_table_opts(internet_gateway_id: Text) -> Tuple[pulumi.ResourceOptions,
         'RouteTableId',
         ['tags', 'routes']
     )
+
+
+def build_userdata(  # noqa: WPS211
+        instance_name: Text,
+        minion_keys: OLSaltStackMinion,
+        minion_roles: List[Text],
+        minion_environment: Text,
+        salt_host: Text,
+        additional_cloud_config: Optional[Dict[Text, Any]] = None,
+        additional_salt_config: Optional[Dict[Text, Text]] = None
+) -> pulumi.Output[Text]:
+    def _build_cloud_config_string(keys) -> Text:  # noqa: WPS430
+        cloud_config = additional_cloud_config or {}
+        # TODO (TMM 2020-09-10): Once the upstream PR is merged move to using the
+        # upstream bootstrap script. https://github.com/saltstack/salt-bootstrap/pull/1498
+        salt_config = {
+            'bootcmd': [
+                'wget -O /tmp/salt_bootstrap.sh https://raw.githubusercontent.com/mitodl/salt-bootstrap/develop/bootstrap-salt.sh',  # noqa: E501
+                'chmod +x /tmp/salt_bootstrap.sh',
+                'sh /tmp/salt_bootstrap.sh -U -N -z'
```

Comment:
Why isn't this pointing to the salt-master?

Reply:
Because the next block in the cloud-init handles setting up the configuration. This is just for setting up the package repo so that the most recent version of Salt gets installed. That's what the `-z` flag does which I added in a PR yesterday.